test/selectBuildConfigs.test.ts

```diff
@@ -0,0 +1,157 @@
+});
+
+const workflows: string[] = [
+`jobs:
+  build_and_deploy_job:
+    steps:
+      - uses: Azure/static-web-apps-deploy@v0.0.1-preview
+        id: builddeploy
+        with:
+          app_location: "/"
+          api_location: ""
+          output_location: ""`,
+
+`jobs:
+  build_and_deploy_job:
+    steps:
+      - uses: Azure/static-web-apps-deploy@v0.0.1-preview
+        id: builddeploy
+        with:
+          app_location: "app/location"
+          api_location: 'api/location'
+          app_artifact_location: app/artifact/location`,
+
+`jobs:
+  build_and_deploy_job1:
+    steps:
+      - uses: Azure/static-web-apps-deploy@v0.0.1-preview
+        id: builddeploy
+        with:
+            app_location: "src"
+            api_location: "api"
+            output_location: "build"
+
+  build_and_deploy_job2:
+    steps:
+      - uses: Azure/static-web-apps-deploy@v0.0.1-preview
+        id: builddeploy
+        with:
+          app_location: "src"
+          api_location: "api"
+          output_location: "build"`,
+
+`name: Azure Static Web Apps CI/CD
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - master
+
+jobs:
+  build_and_deploy_job:
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    runs-on: ubuntu-latest
+    name: Build and Deploy Job
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Build And Deploy
+        id: builddeploy
+        uses: Azure/static-web-apps-deploy@v0.0.1-preview
+        with:
+          azure_static_web_apps_api_token: $\{{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_AMBITIOUS_ROCK_0D992521E }}
+          repo_token: $\{{ secrets.GITHUB_TOKEN }} # Used for Github integrations (i.e. PR comments)
+          action: "upload"
+          ###### Repository/Build Configurations - These values can be configured to match your app requirements. ######
+          # For more information regarding Static Web App workflow configurations, please visit: https://aka.ms/swaworkflowconfig
+          app_location: "super/long/path/to/app/location"		# There are tabs before this comment
+          api_location: 'single/quotes'                         #There are spaces before this comment
+          output_location: output/location with/spaces # There is a single space before this comment
+          ###### End of Repository/Build Configurations ######
+
+  close_pull_request_job:
+    if: github.event_name == 'pull_request' && github.event.action == 'closed'
+    runs-on: ubuntu-latest
+    name: Close Pull Request Job
+    steps:
+      - name: Close Pull Request
+        id: closepullrequest
+        uses: Azure/static-web-apps-deploy@v0.0.1-preview
+        with:
+          azure_static_web_apps_api_token: $\{{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_AMBITIOUS_ROCK_0D992521E }}
+          action: "close"
+
+`];
```

Comment:
Could we try location paths with special characters and maybe some foreign characters? (Chinese or Korean or something seems plausible)